include/openssl/ssl.h

```diff
@@ -685,11 +682,11 @@ void SSL_CTX_sess_set_get_cb(SSL_CTX *ctx,
                                                              *data, int len,
                                                              int *copy));
 SSL_SESSION *(*SSL_CTX_sess_get_get_cb(SSL_CTX *ctx)) (struct ssl_st *ssl,
-                                                       const unsigned char *data,
-                                                       int len, int *copy);
+                                                       const unsigned char
+                                                       *data, int len,
+                                                       int *copy);
```

Comment:
For these sort of callback getters, I'd almost consider a more radical solution that indents the getter arguments only a few columns more than, say, the start of the getter function name.  But definitely wait for a second opinion before implementing that!

Reply:
A better solution `struct ssl_st* ->   SSL*` ...
plus a typedef on callback functions  ... 

I did not dig the origins of this kind of declarations ...